### PR TITLE
must add "sudo" otherwise the command will failed with error "#define…

### PR DESCRIPTION
**Pull request by wangqiang1588**
…    EPERM         1    /＊ Operation not permitted ＊/"

### CODE REVIEW

**Comment by rabbah** [comment]
Where are you getting permission error? Sudo should not be required.

**Comment by wangqiang1588** [comment]
ubuntu default user does not have root permission , if you do not getting permission error ,your current user should be root

**Comment by wangqiang1588** [comment]
it is better to add sudo ,even root

File: tools/ubuntu-setup/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Follow instructions [tools/db/README.md](../db/README.md) on how to configure a
 
   ```
   cd <home_openwhisk>
-  ./gradlew distDocker
+  sudo ./gradlew distDocker
```
Comment (rabbah) [inline tools/ubuntu-setup/README.md]: i don't think we should make this change - building and deploying in a local environment should not require privilege escalation.

**Comment by rabbah** [comment]
Going to close this per my comment above.

**Pull request closed without merging**